.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
> .idea

Please remove IDE related files from the Git.
Configure "global .gitignore" for handling that in the future.

Rebase / squash all commits into one after that (to cleanup Git history).